pkg/pd/client.go

```diff
@@ -38,18 +44,22 @@ const (
 type Client struct {
 	httpScheme   string
 	baseURL      string
+	specificURL  string
 	httpClient   *httpc.Client
 	lifecycleCtx context.Context
 	timeout      time.Duration
+	cache        *ttlcache.Cache
 }
 
 func NewPDClient(lc fx.Lifecycle, httpClient *httpc.Client, config *config.Config) *Client {
 	client := &Client{
 		httpClient:   httpClient,
 		httpScheme:   config.GetClusterHTTPScheme(),
 		baseURL:      config.PDEndPoint,
+		specificURL:  "",
```

Comment:
What is this hack?

Reply:
Updated. This seems like an unnecessary hack.